### DISCUSSION TITLE
modelLabel is converted to uppercase letter

### DISCUSSION THREAD

**Discussion by ekrist1** [filamentphp/filament#7915] [General]
When you customiz the model label, the word is converted to an uppercase letter. This will be correct in the English language, but may not fit other languages. From what i can see, filament uses the Laravel helper function _headline_. Is there a reason why the model label is converted, instead of using the spelled word?

`protected static ?string $modelLabel = 'cliente';` => returns Cliente

// [packages/panels/src/Resources/Pages/CreateRecord.php](https://github.com/filamentphp/filament/blob/d018f51e8d79feab6a65f6ead8bb9fc3e93d64b9/packages/panels/src/Resources/Pages/CreateRecord.php#L247)

`return __('filament-panels::resources/pages/create-record.title', [
            'label' => Str::headline(static::getResource()::getModelLabel()),
        ]);`

**Comment by buderdene** [comment]
File map : Vendor/Filament/Filament/scr/Resource/Resource/php


public static function getTitleCasePluralModelLabel(): string
    {
        if (! static::hasTitleCaseModelLabel()) {
            return static::getPluralModelLabel();
        }
        return static::getPluralModelLabel();
        //return Str::ucwords(static::getPluralModelLabel());
    }

**Reply by micraux** [reply]
Not recommended. Try to avoid modifying the vendor files, they could be replaced in the next version!

See here: https://github.com/filamentphp/filament/discussions/10384
